docs/Privacy/Private-Transactions/eeajs.md

```diff
@@ -0,0 +1,55 @@
+description: web3.js-eea Client Library 
+<!--- END of page meta data -->
+
+# web3.js-eea Client Library
+
+The [web3.js-eea library](https://github.com/PegaSysEng/eeajs) adds an additional property to your web3 
+instance by extending [web3](https://github.com/ethereum/web3.js/). Use the library to create and send 
+RLP-encoded transactions using JSON-RPC.
+
+!!! note
+    web3.js-eea supports JSON-RPC over HTTP only. 
+
+## Prerequisites
+
+- [Node.js (version > 10)](https://nodejs.org/en/download/)  
+- [web3.js 1.0.0](https://web3js.readthedocs.io/en/1.0/getting-started.html)
+
```

Comment:
This will get pulled in when you add web3-eea library.

Reply:
Fixed (ie, removed web.js as a pre-req)